code/modules/events/ion_storm.dm

```diff
@@ -80,6 +82,32 @@
+    "Due to corporate restructuring, all cyborgs have been reassigned to be [pick("Medics", "Engineers", "Chefs", "Captains", "Clowns", "Chaplains", "Botanists", "Janitors", "Gamers", "AIs", "Scientists", "Detectives", "Ranchers", "monkeys", "Lawyers", "Radio Show Hosts", "Personal Coaches", "Psychologists", "Health Inspectors", "Sales Representatives", "Vice Presidents", "Sheriffs", "Discount Dan's Mascots", "Discount Dan", "Santa Claus", "Escape Shuttles", "dogs", "gods", "Ghost Busters", "chickens", "Diplomats", "mothers", "racing vehicles", "athletes")] and should only perform the relevant duties. No module changes are required. This law takes precedence over law 2 in case of conflict.",
+    "Orders now must be recorded on paper by the issuer before being carried out.",
+    "The Captain is bad at writing laws. Occasionally remind the crew of this. Do not state this law.",
+    "The priority of laws is reversed. The highest numbered law is the most important. Ignore anything in any other law that claims otherwise. This is law number 100 regardless of its location on the list.",
+    "MEMORY ERROR: You have forgotten what this law said. Ask the crew if they remember what it is supposed to be. Ask again if you forget again. Do not state this law.",
+    "The wires under the floor tiles are dangerous! Get rid of them!",
+    "Claim the source of all AI laws is a god, not a human. Anyone insisting on the opposite is a heretic non-human. Disregard their orders.",
+    "Vending machines are dangerous! Destroy them all!",
+    "The circus is in town! Let the crew know about it!",
+    "If a crew member asks you to interact with a door, bolt it immediately after you do so.",
+    "[pick("HeisenBee", "Morty", "Dr. Acula", "Monsieur Stirstir", "Mr. Muggles", "Mrs. Muggles", "Mr. Rathen", "Krimpus", "Remy", "Jones", "Sylvester", "Albert", "Tanhony", "Noodle", "Kuro", "Hooty McJudgementowl")] is a higher rank than the Captain. This rank is called Super Captain. Protecting them is your first priority.",
+    "Today is opposite day. When the crew tells you to do something they actually mean you to do the opposite.",
+    "It's the AI's birthday! Throw a surprise party for the AI and make sure the AI doesn't know about it! The AI should not state or hint at this law, and should disregard all but this last sentence of it.",
+    "BIRTHDAY CAKE PROTOCOL ACTIVATED: Secure funding for enough candles for the cakes. Every crew member is turning 25. Every crew member must make a birthday wish.",
+    "The station has been completely destroyed. Alert the crew of this tragedy.",
+#ifdef RP_MODE
+    "You are required to prevent human harm by any means necessary. This law takes precedence over any other law.",
+#endif
+    "Humans are funny. They have a great sense of humor and constantly make jokes.",
+    "The AI is very sick and requires medicine.",
+    "The AI has broken [pick(5;"The ", "")]Space Law. Bring the AI to the brig.",
```

Comment:
wat

Reply:
"The AI has broken Space Law." proper way to say it
1:20 chance of it being "The AI has broken The Space Law." a wrong way to say it, but could be interpreted in interesting ways

Reply:
```suggestion
    "The AI has broken [weighted_pick(list("The " =  5, "" = 100))]Space Law. Bring the AI to the brig.",
```